project/api/models.py

```diff
@@ -11,23 +11,24 @@
 
 
 class EntityManager(PolymorphicManager):
-    """
-    Manager for the Nation model to handle lookups by url_id
-    """
+    """Manager for the Nation model to handle lookups by url_id."""
 
     def get_by_natural_key(self, url_id):
+        """Returns attributes by the given url_id."""
         return self.get(url_id=url_id)
 
 
 class Entity(PolymorphicModel):
-    """
-    Cultural/governmental entity. Serves as foreign key for most Territories
+    """Cultural/governmental entity.
+
+    Serves as foreign key for most Territories.
     """
     objects = EntityManager()
 
     name = models.TextField(
         max_length=100,
-        help_text="Canonical name, should not include any epithets, must be unique",
+        help_text="Canonical name, which should not include any epithets"
```

Comment:
I don't think we need to conform so strictly to `-l 80`. We should leave this on one line imo. Also avoid extra migration.

Reply:
The split is done elsewhere in the same file already, so it makes sense to me for it to be consistent.